install/kubernetes/cilium/README.md

```diff
@@ -63,6 +63,7 @@ contributors across the globe, there is almost always someone available to help.
 | bgp.announce.loadbalancerIP | bool | `false` | Enable allocation and announcement of service LoadBalancer IPs |
 | bgp.enabled | bool | `false` | Enable BGP support inside Cilium; embeds a new ConfigMap for BGP inside cilium-agent and cilium-operator |
 | bpf.clockProbe | bool | `false` |  |
+| bpf.lbExternalClusterIP | bool | `false` | Enable native IP masquerade support in eBPF -- Configure whether direct routing mode should route traffic via host stack (true) or directly and more efficiently out of BPF (false) if the kernel supports it. The latter has the implication that it will also bypass netfilter in the host namespace. -- Configure the eBPF-based TPROXY to reduce reliance on iptables rules for implementing Layer 7 policy. tproxy: true -- Configure the FIB lookup bypass optimization for nodeport reverse NAT handling. lbBypassFIBLookup: true -- Allow cluster external access to ClusterIP services. |
```

Comment:
Looks like this is taking the doc strings starting from the `bpf.masquerade` option down to the (desired) `bpf.lbExternalClusterIP`. I guess we have this result because the values starting from `bpf.masquerade` (included) until `bpf.lbExternalClusterIP` (excluded) are commented.